semgrep-core/src/mlgrep/cli-lib/Error.ml

```diff
@@ -0,0 +1,37 @@
+(*
```

Comment:
Would be nice to have a file somewhere that lists which files have been converted (or half-converted) and which
files remain.